### PR TITLE
Add getOSDescription()

### PR DESCRIPTION
**Pull request by tejohnso**
- 32bit Ubuntu 14.04.4 LTS
 - 64bit Linux 4.6.2-1-ARCH
 - 32bit Microsoft Windows 7 Professional

@ahmedalsudani please review

### CODE REVIEW

File: platform.js [hunk 1]
```diff
@@ -33,6 +33,19 @@ module.exports = {
...
     return childProcess.spawnSync("lsb_release", ["-sr"]).stdout;
   },
+  getLinuxDescription() {
+    return "Linux " + childProcess.spawnSync("uname", ["-r"]).stdout.toString().trim();
+  },
+  getLSBDescription() {
+    return childProcess.spawnSync("lsb_release", ["-dr"]).stdout;
+  },
+  getOSDescription() {
+    return (os.arch() === "ia32" ? "32bit " : "64bit ") +
```
Comment (ahmedalsudani) [inline platform.js]: @tejohnso we should add `arm` handling here

**Comment by tejohnso** [comment]
@ahmedalsudani Can you try platform.getOSDescription() on the pi?

**Comment by tejohnso** [comment]
@ahmedalsudani @fjvallarino tested on win7, win10, archlinux, Ubuntu14.04

**Comment by fjvallarino** [comment]
:+1:

**Comment by ahmedalsudani** [comment]
@tejohnso on the Pi `arm Raspbian GNU/Linux 8.0 (jessie)\n'`

:+1:

**Pull request merged**